src/map/utils/battleutils.cpp

```diff
@@ -2400,6 +2403,9 @@ namespace battleutils
             damage = 0;
         }
 
+        // Handle damage cap mod
```

Comment:
I reckon you don't need the comments everywhere saying this, now that the function is called `CheckAndApplyDamageCap`